tests/unit/test_table.py

```diff
@@ -4027,8 +4026,14 @@ def test_to_geodataframe(self):
         self.assertEqual(df.name.dtype.name, "object")
         self.assertEqual(df.geog.dtype.name, "geometry")
         self.assertIsInstance(df.geog, geopandas.GeoSeries)
-        self.assertEqual(list(map(str, df.area)), ["0.0", "nan", "0.5"])
-        self.assertEqual(list(map(str, df.geog.area)), ["0.0", "nan", "0.5"])
+
+        with warnings.catch_warnings():
+            # Computing the area on a GeoDataFrame that uses a geographic Coordinate
+            # Reference System (CRS) produces a warning that we are not interested in.
+            warnings.filterwarnings("ignore", category=UserWarning)
+            self.assertEqual(list(map(str, df.area)), ["0.0", "nan", "0.5"])
+            self.assertEqual(list(map(str, df.geog.area)), ["0.0", "nan", "0.5"])
```

Comment:
`warnings.filterwarnings` is a global change:  I believe what you want is to trap the warnings for this test case, i.e.:
```suggestion
            with warnings.catch_warnings():
                warnings.filterwarnings("ignore", category=UserWarning)
                self.assertEqual(list(map(str, df.area)), ["0.0", "nan", "0.5"])
                self.assertEqual(list(map(str, df.geog.area)), ["0.0", "nan", "0.5"])
```

Reply:
That's exactly the code in the test? Do you see it differently perhaps?

Reply:
I saw that you used `warnings.catch_warnings()` in the other locations, but somehow I couldn't see it here.